rule.php

```diff
@@ -114,6 +116,19 @@ public static function add_settings_form_fields(mod_quiz_mod_form $quizform, Moo
                 $mform->setDefault($name, $defaults[$name]);
             }
 
+            // Second pass to populate the filemanager with any existing saved self config template.
+            if (is_array($type) && $type[0] == 'filemanager') {
+                $draftitemid = 0;
+                file_prepare_draft_area(
+                    $draftitemid,
+                    $quizform->get_context()->id,
+                    'quizaccess_seb',
+                    $name,
+                    settings_provider::SEB_CONFIG_FILE_ITEMID
```

Comment:
This code supposed to be flexible and could be used for any filemanager in the form. I'd suggest to get rid of settings_provider::SEB_CONFIG_FILE_ITEMID as it adds some confusion. Is it safe to hardcode to 0? Seems like it could be anything and different parts of moodle use different value :thinking: